plugin/vm.go

```diff
@@ -333,3 +333,13 @@ func NewInternalContractContext(contractName string, pluginVM *PluginVM) (contra
 	readOnly := false
 	return contractpb.WrapPluginContext(pluginVM.CreateContractContext(caller, contractAddr, readOnly)), nil
 }
+
+
+func NewInternalStaticContractContext(contractName string, pluginVM *PluginVM) (contractpb.StaticContext, error) {
```

Comment:
Add `readOnly` param to `NewInternalContractContext`, no need to have a separate function for this.